README.md

```diff
@@ -80,6 +80,7 @@ If you want to support IE9, you will need to use a classList pollyfill, like Eli
 * __onHideToolbar__: optional callback that will be called each time the toolbar is actually hidden for this instance of medium-editor.
 * __staticToolbar__: enable/disable the toolbar always displaying in the same location relative to the medium-editor element. Default: false
 * __stickyToolbar__: enable/disable the toolbar "sticking" to the medium-editor element when the page is being scrolled. Default: false
+* __fixToolbar__: enable/disable the toolbar being "fixed" to the top of the page. Use this instead of _stickyToolbar_. Default: false
```

Comment:
This does not use the scroll event to position the toolbar, but instead it's fixed with CSS only. Not really much of a difference I suppose. It centers the toolbar horizontally and keeps it on top regardless of scroll position or the top of the editable element.